profile.json

```diff
@@ -0,0 +1,3 @@
+[
+
+]
```

Comment:
@anthonyshew can we delete this and add it to gitignore? Guessing you ran with `--profile` in the root of this repo

Reply:
🤦 

I won't put it in the gitignore since you can name the file manually (--profile=awesomeprofile.json) but I removed it in #6802.